postgres/lib.w

```diff
@@ -45,16 +42,52 @@ pub class Database {
   }
 }
 
-pub class DatabaseNeon impl IDatabase {
+class DatabaseSim impl IDatabase {
```

Comment:
Maybe put under `db.sim.w`

Reply:
Yes, in other projects we needed to put the interface into a separate file.

I am okay to leave as is

Reply:
Props would have to go in a separate file as well

Reply:
crap